[External API Doc] Added rate limit endpoint section

jiepiao opened the pull request:
### Description

Added rate limit endpoint section.

### Screenshot
![Screen Shot 2022-09-13 at 2 12 35 PM](https://user-images.githubusercontent.com/92333171/190010184-725259a8-f722-4e94-89c3-ee152560bd0c.png)

![Screen Shot 2022-09-13 at 2 13 14 PM](https://user-images.githubusercontent.com/92333171/190010273-d1031582-9aae-4425-8e53-460dfebb2b47.png)

![Screen Shot 2022-09-13 at 2 13 32 PM](https://user-images.githubusercontent.com/92333171/190010321-abde2749-0b5c-419c-b6d3-fc542bc89838.png)

dachan commented:
@jiepiao can we also link to this from the [Rate Limits](https://api-docs.affinity.co/#rate-limits) section and add a mention of it in the [Changelog](https://api-docs.affinity.co/#changelog)?

jiepiao commented:
> @jiepiao can we also link to this from the [Rate Limits](https://api-docs.affinity.co/#rate-limits) section and add a mention of it in the [Changelog](https://api-docs.affinity.co/#changelog)?

Sure, added link and updated changelog. I won't merge this until we enable the feature for all orgs.

dachan reviewed (approved):
Looks good to me, thanks!

marcypark commented:
The rate-limit endpoint header says "whoami"

jiepiao commented:
> The rate-limit endpoint header says "whoami"

Fixed and added monthly organization limit reset description.

marcypark commented on source/includes/_changelog.md:
```diff
@@ -1,5 +1,9 @@
 
+## 2022-09-06
+
+- Added [Rate Limit](#rate-limit) documentation.
```
```suggestion
- Added [Rate Limit](#rate-limit) endpoint and documentation. Moved from a daily to a per minute per user limit. Monthly per account limits remain the same.
```

marcypark commented on source/includes/_rate-limits.md:
```diff
@@ -2,6 +2,8 @@
 
 ## API Rate Limits
 
+For rate limit endpoint, please refer <a href="#rate-limit">Rate Limit</a>.
```
```suggestion
For our rate limit endpoint, please refer to <a href="#rate-limit-endpoint">Rate Limit endpoint</a>.
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+# Rate Limit
+
+The rate limit endpoint allows you to see the monthly organization and per minute per api key limit usage.
```
```suggestion
The rate limit endpoint allows you to see your monthly account-level and per minute user-level API limits and usage.
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+# Rate Limit
+
+The rate limit endpoint allows you to see the monthly organization and per minute per api key limit usage.
+The monthly organization limit resets on at the end of each calendar month.
```
```suggestion
The monthly account-level call limit resets at the end of each calendar month.
```

marcypark commented on source/includes/_rate-limits.md:
```diff
@@ -13,9 +15,9 @@ Current rate limits by plan tier are:
 | Premium       | 100,000         |
 | Enterprise    | Unlimited       |
```
Add a line here too:

This monthly account-level limit resets at the end of each calendar month.

marcypark commented on source/includes/_rate-limits.md:
```diff
@@ -2,6 +2,8 @@
+
 ### Plan Tier Limits
 
 Your account plan tier will limit the number of requests each API key can make per month.
```
```suggestion
Your account plan tier will limit the overall number of requests you can make per month.
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+
+## The Rate Limit Resource
+
+Querying the rate limit endpoint will give information about organization and api key level rate limit usage.
```
```suggestion
The rate limit resource includes information about account (AKA organization)-level and API key-level rate limits and usage.
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+| ------------------ | ------- | -------------------------------------------  |
+| rate               | object  | Type of rate limit.                          |
+| org_monthly        | object  | Monthly rate limit per organization.         |
+| api_key_per_minute | object  | Per minute rate limit per api key.           |
```
```suggestion
| api_key_per_minute | object  | Per minute rate limit per API key.           |
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+| api_key_per_minute | object  | Per minute rate limit per api key.           |
+| limit              | integer | The maximum number of calls for each period. |
+| remaining          | integer | The number of calls remaining before reset.  |
+| reset              | integer | Time in seconds until next time limit reset. |
```
```suggestion
| reset              | integer | Time in seconds until call count resets. |
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+<aside class="notice">
+  <h6>Note</h6>
+  <ul>
+    <li><code>/rate-limit</code> and <code>/auth/whoami</code> endpoints are excluded from rate limiting rules.</li>
```
```suggestion
    <li><code>/rate-limit</code> and <code>/auth/whoami</code> endpoints are exempt from organization-level monthly rate limits.</li>
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+  </ul>
+</aside>
+
+## Get Rate Limit
```
```suggestion
## Get Rate Limit Information
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+
+`GET /rate-limit`
+
+Gets information about the organization and api key level rate limit usage.
```
```suggestion
Querying the rate limit endpoint will yield information about account (AKA organization)-level and API key-level rate limits and usage.
```

marcypark commented on source/includes/_rate-limit-endpoint.md:
```diff
@@ -0,0 +1,82 @@
+
+### Returns
+
+JSON body of data including limit, remaining, reset and used call number.
```
```suggestion
The rate limit resource, a JSON body of data including limits, calls remaining, seconds until reset and call count.
```

marcypark reviewed (commented):
Took the chance to take a look over the whole PR -- Lmk if you'd like to review these on Zoom! It should hopefully be easy just to merge my suggestions straight in.
